dist/index.html

```diff
@@ -5,9 +5,18 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>To Do List</title>
+    <link href="https://fonts.googleapis.com/css?family=Amatic+SC" rel="stylesheet">
 </head>
 <body>
-    
+        <div class="title">
+                <h2>Things To Do</h2>
+                <input type="text" id="myInput" placeholder="Write here something to do...">
+                <button class="add" type="button">Add!</button>
+              </div>
+              
+              <ul id="thingsToDoList">
```

Comment:
Podría ser algo más sencillo como to-do-list, creo que no se pueden poner mayusculas en los nombres de clases e ids